03-components.Rmd

```diff
@@ -1,10 +1,129 @@
 
-# Components of a reproducible analysis 
+# Components of a reproducible analysis
 
 ```{r, include = FALSE}
 ottrpal::set_knitr_image_path()
 ```
 
+In this chapter, we will discuss what components of an analysis make it reproducible.
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Major point!! example image"}
```

Comment:
maybe describe what version control is instead of using a term that people may not know? like changes are tracked over time

Reply:
could also change "understand" to "recognize" apparently some people don't like "understand" for learning objectives